tests/tests/test_passbands/test_passbands.py

```diff
@@ -7,7 +7,7 @@
 path = os.path.dirname(__file__)
 
 
-def test_compute(atm_grids_available=False, atm_path=None, atms=supported_atms.keys()):
+def test_compute(atm_grids_available=False, atm_path=None, atms=supported_atms):
```

Comment:
ah, thanks for catching this; I believe it should be `atms=atm_tables.keys()` instead.

Reply:
nope -- blackbody atms are already included (see the body of the function) and extern_* are only indexed, not computed. I'm actually thinking it might be time to retire extern_*.